admin/install.php

```diff
@@ -246,9 +246,15 @@ function print_test( $p_test_description, $p_result, $p_hard_fail = true, $p_mes
 
 		if( $f_db_type == 'mssql' ) {
 			print_test( 'Checking PHP support for Microsoft SQL Server driver',
-				version_compare( phpversion(), '5.3' ) < 0, true,
+				BAD, true,
 				'mssql driver is no longer supported in PHP >= 5.3, please use mssqlnative instead' );
 		}
+
+		if( $f_db_type == 'mysql' ) {
+			print_test( 'Checking PHP support for MySQL driver',
+				BAD, true,
+				'mysql driver is deprecated in PHP >= 5.5, please use mysqli instead' );
```

Comment:
I would add that we no longer support it.

Reply:
Changed the message.